dart/generate_map/README.md

```diff
@@ -0,0 +1,72 @@
+# 🗺️ Generate Map
+
+A Dart Cloud Function that generates a static map base64 encoded image at given latitude and longitute.
+
+_Example input 1:_
+
+```json
+{
+    "lat": "50", 
+    "lng": "50"
+}
+```
+
+_Example output 1:_
+
+
+```json
+{
+    "success": "true",
+    "image": "9j..longString.dasdsdasd/sadas/d/asd/asdasfasasfssdafad="
+}
+```
+_Example input 2:_
+
+```json
+{
+    "lat": "50", 
+    "lng": "-270"
+}
+```
+
+_Example output 2:_
+
+
+```json
+{
+    "success": "false",
+    "message": "The longitude must be between -180 and 180"
+}
+```
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud function:
+
+- **MAPQUEST_API_KEY** - Your MAP Quest API KEY 
+
+## 🚀 Deployment
+
+1. Clone this repository, and enter this function folder:
+
+```
+$ git clone https://github.com/open-runtimes/examples.git && cd examples
+$ cd dart/generate-map
+```
+
+2. Enter this function folder and build the code:
+```
+docker run -e INTERNAL_RUNTIME_ENTRYPOINT=lib/main.dart --rm --interactive --tty --volume $PWD:/usr/code openruntimes/dart:2.16 sh /usr/local/src/build.sh
+```
+As a result, a `code.tar.gz` file will be generated.
+
+3. Start the Open Runtime:
+```
+docker run -p 3000:3000 -e INTERNAL_RUNTIME_KEY=secret-key --rm --interactive --tty --volume $PWD/code.tar.gz:/tmp/code.tar.gz:ro openruntimes/dart:2.16 sh /usr/local/src/start.sh
```

Comment:
Would you please update the docker image to our latest version?

````suggestion
docker run -e INTERNAL_RUNTIME_ENTRYPOINT=lib/main.dart --rm --interactive --tty --volume $PWD:/usr/code openruntimes/dart:v2-2.16 sh /usr/local/src/build.sh
```
As a result, a `code.tar.gz` file will be generated.

3. Start the Open Runtime:
```
docker run -p 3000:3000 -e INTERNAL_RUNTIME_KEY=secret-key --rm --interactive --tty --volume $PWD/code.tar.gz:/tmp/code.tar.gz:ro openruntimes/dart:v2-2.16 sh /usr/local/src/start.sh
````